dist_tables/ddl.rst

```diff
@@ -51,7 +51,11 @@ You are now ready to insert data into the distributed table and run queries on i
 Reference Tables
 ~~~~~~~~~~~~~~~~
 
-The above method distributes tables into multiple horizontal shards, but it's also possible to distribute tables into a single shard and replicate it to every worker node. Tables distributed this way are called *reference tables.*  They are typically small non-partitioned tables which we want to locally join with other tables on any worker. One US-centric example is information about states.
+The above method distributes tables into multiple horizontal shards, but another possibility is distributing tables into a single shard and replicating the shard to every worker node.
+
+Tables distributed this way are called *reference tables.* They are used to store data that needs to be frequently accessed by multiple nodes in a cluster. In multi-tenant apps this means data shared between tenants or data that is not naturally associated with a single tenant.
```

Comment:
The SE Doc we listed said adding a section like "When to use Reference Tables". I don't know if there is more to add than this paragraph. @samay-sharma Any thoughts?

Reply:
A few things on "When to use reference tables"

(a) Smaller tables which need to join with larger distributed tables.
(b) Tables which do not have the tenant column or aren't associated with the tenant. (In some cases, users might do it even for tables which are associated with tenant if they don't have tenant id to reduce migration effort.)
(c) Tables which need unique constraints across multiple columns and are small enough.

Reply:
Thanks @samay-sharma  
@begriffs Could you add these points (bullet listed as Samay wrote up) to the paragraph?

Reply:
@begriffs Is this PR updated with Samay's comments? Thanks